src/deprecated/deprecated.js

```diff
@@ -465,6 +464,19 @@ Object.defineProperties(Texture.prototype, {
 
 // SCENE
 
+class DepthMaterial extends Material {
+    updateShader(device) {
+        Debug.deprecated("pc.DepthMaterial is deprecated.");
+        const options = {
+            skin: !!this.meshInstances[0].skinInstance
+        };
+        const library = device.getProgramLibrary();
+        this.shader = library.getProgram('depth', options);
```

Comment:
Does this even work? Where is the 'depth' program registered?

Reply:
![image](https://user-images.githubusercontent.com/697563/155709345-ef83d427-8341-4bef-92f4-e0351e7d0aba.png)

Reply:
you're right, none of that exists anymore. I'll delete DepthMaterial completely.